controller/events.go

```diff
@@ -1,16 +1,199 @@
 package main
 
 import (
+	"database/sql"
```

Comment:
This import should be `github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-sql`